pkgs/development/python-modules/cli-ui/default.nix

```
@@ -0,0 +1,52 @@
+{
+  lib,
+  python3Packages,
+  fetchPypi,
+  pytestCheckHook,
+  pythonRelaxDepsHook,
+  pythonOlder,
+  poetry-core,
+}:
+python3Packages.buildPythonPackage rec {
+  pname = "cli-ui";
+  version = "0.17.2";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-L2flDPR052rRYMPmYLutmL+LjfuNhHdl86Jht+E8Bfo=";
+  };
+
+  pythonRelaxDeps = [
+    "tabulate"
+  ];
+
+  nativeBuildInputs = [
+    pythonRelaxDepsHook
+  ];
+
+  build-system = [
+    poetry-core
+  ];
+
+  dependencies = with python3Packages; [
```

```suggestion
  dependencies = [
```

Please pass those dependencies in the top level arguments. You should also remove `python3Packages` from the top level arguments.

---

Sorry ignore me, this comment was for the other file actually.